test/custom/incremental_strategies/test_incremental_strategies.py

```diff
@@ -0,0 +1,102 @@
+from cProfile import run
+from test.custom.base import DBTSparkIntegrationTest, use_profile
+import dbt.exceptions
+
+
+class TestIncrementalStrategies(DBTSparkIntegrationTest):
+    @property
+    def schema(self):
+        return "incremental_strategies"
+
+    @property
+    def project_config(self):
+        return {
+            'seeds': {
+                'quote_columns': False,
+            },
+        }
+
+    def seed_and_run_twice(self):
+        self.run_dbt(["seed"])
+        self.run_dbt(["run"])
+        self.run_dbt(["run"])
+
+
+class TestDefaultAppend(TestIncrementalStrategies):
+    @property
+    def models(self):
+        return "models"
+        
+    def run_and_test(self):
+        self.seed_and_run_twice()
+        self.assertTablesEqual("default_append", "expected_append")
+
+    @use_profile("apache_spark")
+    def test_default_append_apache_spark(self):
+        self.run_and_test()
+
+    @use_profile("databricks_cluster")
+    def test_default_append_databricks_cluster(self):
+        self.run_and_test()
+
+
+class TestInsertOverwrite(TestIncrementalStrategies):
+    @property
+    def models(self):
+        return "models_insert_overwrite"
+
+    def run_and_test(self):
+        self.seed_and_run_twice()
+        self.assertTablesEqual(
+            "insert_overwrite_no_partitions", "expected_overwrite")
+        self.assertTablesEqual(
+            "insert_overwrite_partitions", "expected_upsert")
```

Comment:
🤦

Reply:
You're good!!! For a very dumb reason, this _worked_ in Databricks when it really shouldn't have. I fixed via:
<img width="641" alt="Screen Shot 2021-02-19 at 2 50 36 PM" src="https://user-images.githubusercontent.com/13897643/108512661-d6077680-72c1-11eb-8160-3f08a246a977.png">